main.go

```diff
@@ -0,0 +1,242 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-steputils/tools"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/google/go-querystring/query"
+	"github.com/pkg/errors"
+)
+
+const (
+	baseURL                   = "https://api.bitrise.io/v0.1"
+	buildTypeTag              = "tag"
+	buildTypePush             = "push"
+	buildTypeManual           = "manual"
+	buildTypePullRequest      = "pull-request"
+	envKeyPreviousBuildStatus = "PREVIOUS_BUILD_STATUS"
+	envKeyBuildStatusChanged  = "BUILD_STATUS_CHANGED"
+	statusTextSuccessfulBuild = "success"
+)
+
+type config struct {
+	AppSlug             string          `env:"BITRISE_APP_SLUG,required"`
+	BuildSlug           string          `env:"BITRISE_BUILD_SLUG,required"`
+	BuildStatus         string          `env:"BITRISE_BUILD_STATUS,required"`
+	PreviousBuildStatus string          `env:"PREVIOUS_BUILD_STATUS"`
+	AccessToken         stepconf.Secret `env:"access_token,required"`
```

Comment:
What is the reason for this env being lowercase?

Reply:
others are envs we exported or exported on the VM by default, this one is a step input env